lib/types.ts

```
@@ -22,56 +22,58 @@
+  "kinde.m2mToken"?: {
     /**
-     * Add URL tooling
+     * {boolean} reset all claims to default value on workflow start, default is false
      */
-    url?: {};
+    resetClaims?: boolean;
   };
+  /**
+   * Exposes the console methods to the workflow
+   */
+  console?: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)